README.md

```diff
@@ -1,3 +1,8 @@
+
+<p align="center">
```

Comment:
Why did you choose to work with html and not pure markdown?

![Smarticle](https://github.com/redhat-beyond/Smarticle/blob/main/documentations/logo/Smarticle_logo_prev_ui.png){ width="800" height="600" style="display: block; margin: 0 auto" }

Reply:
Acutely, I tried it before, this lines doesn't work for some reason.
I tried another way, to add a CSS file, but In my opinion it is too much to create a new file just centering the image.

I searched examples to insert an image to markdown file, and most of them use html parts, like [here](https://github.com/ElectricRCAircraftGuy/eRCaGuy_hello_world/blob/master/markdown/github_readme_center_and_align_images.md).